src/openrct2/actions/RideDemolishAction.hpp

```diff
@@ -339,8 +339,8 @@ DEFINE_GAME_ACTION(RideDemolishAction, GAME_COMMAND_DEMOLISH_RIDE, GameActionRes
 
             for (Direction dir : ALL_DIRECTIONS)
             {
-                const CoordsXY& off = DirOffsets[dir];
-                money32 removePrice = MazeRemoveTrack(location.x + off.x, location.y + off.y, location.z, dir);
+                const CoordsXYZ& off = { DirOffsets[dir], 0 };
```

Comment:
Sorry, made a mistake. It should have been 
```suggestion
                const CoordsXYZ off = { DirOffsets[dir], 0 };
```